openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildConfigOperationsImpl.java

```diff
@@ -235,7 +235,10 @@ public void writeTo(BufferedSink sink) throws IOException {
         }
       };
 
-      OkHttpClient newClient = client.newBuilder().readTimeout(timeout, timeoutUnit).build();
+      OkHttpClient newClient = client.newBuilder()
+        .readTimeout(timeout, timeoutUnit)
+        .writeTimeout(timeout, timeoutUnit)
```

Comment:
It may be that I messed up the `withTimeout` DSL - IIRC `instantiateBinary` should be terminal.

@iocanel Do you have some time to look at this? You are Mr DSL after all...